docs/replit-prompts/CitationsModule_Integration_Prompt.md

```
@@ -0,0 +1,311 @@
+
+| Task | Owner | Status |
+|------|-------|--------|
+| Citation components | AVR | ✅ Complete |
+| Citation hooks/service | AVR | ✅ Complete |
+| CitationsModule wrapper | **Sakthi** | ⏳ To create |
+| Editorial Services shell | AVR | ⏳ Feb 2 |
+| Route integration | AVR | ⏳ After shell |
+
+---
+
+*Created: February 1, 2026*
```

markdownlint-cli2 (0.20.0)

[warning] Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)